modules/nf-core/deeptools/multibamsummary/main.nf

```diff
@@ -33,4 +33,14 @@ process DEEPTOOLS_MULTIBAMSUMMARY {
         deeptools: \$(multiBamSummary --version | sed -e "s/multiBamSummary //g")
     END_VERSIONS
     """
+
+    stub:
+    """
+    touch all_bam.bamSummary.npz
```

Comment:
This module should be using prefix? It can default to `all_bam`, but there is no way otherwise to get a different named file.